packages/autocomplete-core/src/getPropGetters.ts

```diff
@@ -103,7 +103,7 @@ export function getPropGetters<TItem, TEvent, TMouseEvent, TKeyboardEvent>({
       onSubmit: (event) => {
         ((event as unknown) as Event).preventDefault();
 
-        props.onSubmit({
+        const onSubmitPayload = {
```

Comment:
We use the vocabulary "params" in the codebase so we can go with `onSubmitParams` (same for next occurrences).

Reply:
done https://github.com/algolia/autocomplete.js/pull/316/commits/90fc79061c0beeb95cceedd446d8f1e2445a8b21